core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3678,8 +3680,8 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
         try (CloseableResource<UnderFileSystem> ufsResource = resolution.acquireUfsResource()) {
           UnderFileSystem ufs = ufsResource.get();
           if (ufs.isObjectStorage()) {
-            LOG.debug("setOwner/setMode is not supported to object storage UFS via Alluxio. "
-                + "UFS: " + ufsUri + ". This has no effect on the underlying object.");
+            SAMPLING_LOG.debug("setOwner/setMode is not supported to object storage UFS via "
+                + "Alluxio. UFS: {}. This has no effect on the underlying object.", ufsUri);
```

Comment:
Will this help in the case `ufsUri` is different? Also this is not part of the original change so maybe we want to separate this out (or is the sampling logger change not cherry-pickable anyway)?